viewer/src/main/java/nl/b3p/viewer/stripes/FeatureReportActionBean.java

```diff
@@ -139,7 +139,7 @@ public Resolution print() throws URISyntaxException, IOException, Exception {
             FilterFactory2 ff = CommonFactoryFinder.getFilterFactory2();
             FeatureSource fs = layer.getFeatureType().openGeoToolsFeatureSource(TIMEOUT);
             String geomAttribute = fs.getSchema().getGeometryDescriptor().getLocalName();
-            Filter fidFilter = ff.id(new FeatureIdImpl(this.fid));
+            Filter fidFilter = ff.id(new FeatureIdImpl(this.fid.substring(this.fid.indexOf(".")+1)));
```

Comment:
so what happens when typename contains a dot?

use `this.fid.replace(layer.getFeatureType().getTypeName() + ".", "")`

has the view been properly registered in the metadata table?

Reply:
"so what happens when typename contains a dot?"
We should name-and-shame that person... 
I'll edit it so it'll use lastIndexOf...

View is present in gt_pk_metadata table...

Reply:
I still think it's better to use `this.fid.replace(layer.getFeatureType().getTypeName() + ".", "")` instead of just getting the part of the string after the last dot